pkg/operators/provisioning/repo_operator.go

```diff
@@ -98,16 +106,29 @@ func RunRepoController(deps server.OperatorDependencies) error {
 
 type repoControllerConfig struct {
 	provisioningControllerConfig
-	workerCount int
+	workerCount         int
+	allowedTargets      []string
+	allowImageRendering bool
+	minSyncInterval     time.Duration
 }
 
 func getRepoControllerConfig(cfg *setting.Cfg, registry prometheus.Registerer) (*repoControllerConfig, error) {
 	controllerCfg, err := setupFromConfig(cfg, registry)
 	if err != nil {
 		return nil, err
 	}
+
+	allowedTargets := []string{}
```

Comment:
these defaults are for when running the operator as a separate target only. the original defaults are still the same for grafana itself